radiology.csl

```diff
@@ -180,6 +180,11 @@
     <choose>
       <if type="article-journal article-magazine" match="any">
         <text variable="volume"/>
+        <choose>
+          <if match="any" variable="issue">
+            <text variable="issue" prefix="(" suffix=")"/>
+          </if>
+        </choose>
```

Comment:
The whole `<choose/>` block is unnecessary here. You can just use `<text variable="issue" prefix="(" suffix=")"/>` instead. Can you update your pull request?